python/lsst/obs/base/cli/cmd/commands.py

```diff
@@ -67,12 +67,12 @@ def define_visits(*args, **kwargs):
 
 @click.command(short_help="Ingest raw frames.")
 @repo_argument(required=True)
+@locations_argument(help="LOCATIONS specifies files to ingest and/or locations to search for files.",
+                    required=True)
+@regex_option(help="Regex string used to find files in directories listed in LOCATIONS.")
```

Comment:
I think you need to somehow set the override regex here to be the one that recognizes FITS files.